### ISSUE TITLE
Fix tab orders (KN.2)

### ISSUE THREAD

**Issue by droidmonkey** [keepassxreboot/keepassxc#3164]

**Comment by mojoBrendan** [comment]
Until the last update tab from the search field would take you to the results. Now it's shift-tab

tab from the search field now seems to lose focus altogether so you can't recover by shift-tab twice as you'd expect.

**Comment by mojoBrendan** [comment]
```
KeePassXC - Version 2.4.1
Revision: 7bafe65

Qt 5.5.1
Debugging mode is disabled.

Operating system: Ubuntu 16.04.6 LTS
CPU architecture: x86_64
Kernel: linux 4.15.0-50-generic

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey

Cryptographic libraries:
 libgcrypt 1.8.1
```

**Comment by droidmonkey** [comment]
Yes I think Qt 5.12 introduced some serious tab ordering issues. I cannot get it to work no matter what I do.

**Comment by mojoBrendan** [comment]
Sorry I didn't realize you created the issue as well, just saw that it had no text and was trying to add some context.

Too bad about the Qt changes, many years of muscle memory to rewire :disappointed:
